src/test/controllers/illness/who.was.ill.controller.spec.unit.ts

```diff
@@ -50,6 +61,54 @@ describe("who was ill url tests", () => {
     expect(mockGetCurrentReason).toBeCalled();
   });
 
+  it("should find who was ill page with get existing director", async () => {
+    mockGetCurrentReason.prototype.constructor.mockImplementation(() => reason("Company director or officer"));
+    const res = await request(app)
+      .get(pageURLs.EXTENSIONS_REASON_ILLNESS)
+      .set("Referer", "/")
+      .set("Cookie", [`${COOKIE_NAME}=123`]);
+    expect(res.status).toEqual(200);
+    expect(stringify(res)).toContain("Company%20director%20or%20officer%22%20checked");
```

Comment:
instead of stringify(res), try expect(res.text).toContain("Company director or officer checked") - you should be able to do it with the encoded spaces then (maybe)